<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TOMAS MORALES</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_crear">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GUARDAR_CURSO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_codigo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>CODIGO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>NOMBRE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CREDITOS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_creditos">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_nombre">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OBLIGATORIO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PRE- REQUISITOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CURSOS QUE PUEDES LLEVAR:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_actualizar">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>420</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar_Cursos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_serializar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Serializar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>APROBADO</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Lista_Cursos</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_buscar">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>430</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 127);</string>
    </property>
    <property name="text">
     <string>BUSCAR</string>
    </property>
   </widget>
   <widget class="QComboBox" name="txt_obligatorio">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <item>
     <property name="text">
      <string>si</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>no</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="txt_aprobado">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>no</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>si</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_editar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar_Modificaciones</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="areaPre">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_eliminar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
background-color: rgb(255, 73, 73);</string>
    </property>
    <property name="text">
     <string>Eliminar Curso</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cr.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_cr">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="areaCursos">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>61</y>
      <width>551</width>
      <height>341</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>codigo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>creditos</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>obligatorio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cr. necesarios</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="txt_ingresarCod">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_busqueda_codigo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>460</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 127);</string>
    </property>
    <property name="text">
     <string>Busqueda por codigo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_creditos">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CREDITOS ACTUALES: </string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
